### PR TITLE
OSDOCS-5752

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-5752](https://issues.redhat.com//browse/OSDOCS-5752)

Version(s):
4.12+
Issue:
https://issues.redhat.com/browse/OSDOCS-5752

Link to docs preview:
https://59174--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-limits-scalability.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jneczypor** [comment]
Hi @yuwang-RH! Can you take a look at this PR for me please? Thanks!

**Comment by yuwang-RH** [comment]
LGTM~

**Comment by jneczypor** [comment]
Hi @arendej! Can I please get a PM review for this PR? Thanks so much!

File: modules/sd-planning-cluster-maximums-environment.adoc [hunk 1]
```diff
@@ -50,7 +50,7 @@ The following table lists the OpenShift Container Platform environment and confi
 |===
 [.small]
 --
-1. io1 disks are used for control plane/etcd nodes because etcd is I/O intensive and latency sensitive. A greater number of IOPS can be required, depending on usage.
+1. io1 disks are used for control plane/etcd nodes in all versions prior to 4.10.0.
```
Comment (arendej) [inline modules/sd-planning-cluster-maximums-environment.adoc:53]: in all versions prior to 4.10.0
to
in all versions prior to 4.10

**Review by tmalove — APPROVED** [review]

**Review by arendej — APPROVED** [review]
lgtm

**Pull request merged**